misc/output_test.py

```diff
@@ -162,6 +162,30 @@ def test_tool_inputs(self):
 out2
 ''')
 
+    def test_tabs_indent(self):
+        content = '''
```

Comment:
nit: It is really difficult to see the tabs in this input. I suggest you use explicit `<TAB>` markers and `str.replace()` to replace them with real tabs, as in:

```
    # For clarity, tabs are written as <TAB> in the literal string below, then
    # replaced with a real "\t" value.
    content = '''
    rule exec
    <TAB>command = $cmd
    ....
    """.replace("<TAB>", "\t")
```